urlParamHook.js

```diff
@@ -1,7 +1,6 @@
 const rlite = require('rlite-router')
 
-const onNonMatchingPath = () => false
-const returnParams = params => params
+const onNonMatchingPath = () => ({ matches: false })
+const returnParams = params => ({ matches: true, ...params })
```

Comment:
This is the primary code change. Other changes are made to reflect this change.

Reply:
What would this look like in the case of the root? like `/`

Reply:
If it didn't match (which I'm not sure ever could happen on a hosted page):
```
{ matches: false }
```
If it did match, it would be `matches: true` and any query parameters (no path parameters in this case).